packages/magellan-mocha-plugin/test_support/coffee/.mocharc.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
-	require: [ 'coffeescript/register' ],
+	require: [ 'coffee-script/register' ],
```

Comment:
This was using the package `coffeescript` but `magellan-mocha-plugin` declares `coffee-script`. 

So why did it work before? Because `grunt` was bringing in `coffeescript` as a transitive dependency, so this package was working by accident.